backend/README.md

```diff
@@ -34,13 +34,19 @@ To start the backend locally, use the following command:
 pdm run dev
 ```
 
-Note: If you wish to run the backend outside of Docker (e.g., for local development), you'll need to set up a PostgreSQL database. Ensure your database configuration is set properly in the environment variables before running the project. For the time being, the recommended approach for local development using the database is to use the docker compose Postgres instance with your local dev backend.
+Note: If you wish to run the backend outside of Docker (e.g., for local development), you'll need to set up a PostgreSQL database. Ensure your database configuration is set properly in the environment variables before running the project.
```

Comment:
add some note about setting up the readme properly prior to running this command